### PR TITLE
Create ruleset: Shouldianswer.com

### PR DESCRIPTION
**Pull request by injust**

### CODE REVIEW

File: src/chrome/content/rules/Shouldianswer.com.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	<rule from="^http://odebractelefon\.pl/"
+		to="https://www.odebractelefon.pl/" />
+	<rule from="^http://responderono\.es/"
+		to="https://www.responderono.es/" />
+	<rule from="^http://shouldianswer\.co\.uk/"
+		to="https://www.shouldianswer.co.uk/" />
+	<rule from="^http://shouldianswer\.com/"
+		to="https://www.shouldianswer.com/" />
+	<rule from="^http://sollichannehmen\.de/"
+		to="https://www.sollichannehmen.de/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Shouldianswer.com.xml]:
```
shouldianswer.net
www.shouldianswer.net
au.shouldianswer.net
ar.shouldianswer.net
at.shouldianswer.net
be.shouldianswer.net
br.shouldianswer.net
co.shouldianswer.net
ci.shouldianswer.net
cn.shouldianswer.net
cd.shouldianswer.net
cl.shouldianswer.net
dk.shouldianswer.net
fi.shouldianswer.net
hu.shouldianswer.net
in.shouldianswer.net
id.shouldianswer.net
ie.shouldianswer.net
jp.shouldianswer.net
ke.shouldianswer.net
mx.shouldianswer.net
nl.shouldianswer.net
nz.shouldianswer.net
ng.shouldianswer.net
no.shouldianswer.net
pe.shouldianswer.net
ph.shouldianswer.net
pt.shouldianswer.net
sn.shouldianswer.net
za.shouldianswer.net
se.shouldianswer.net
ch.shouldianswer.net
ve.shouldianswer.net

File: src/chrome/content/rules/Shouldianswer.com.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		- muzutozvednout.cz, equivalent to www.muzutozvednout.cz
+		- neberitrubku.ru, equivalent to www.neberitrubku.ru
+		- odebractelefon.pl, equivalent to www.odebractelefon.pl
+		- responderono.es, equivalent to www.responderono.es
+		- shouldianswer.co.uk, equivalent to www.shouldianswer.co.uk
+		- shouldianswer.com, equivalent to www.shouldianswer.com
+		- sollichannehmen.de, equivalent to www.sollichannehmen.de
+-->
+
+<ruleset name="Shouldianswer.com">
```
Comment (J0WI) [inline src/chrome/content/rules/Shouldianswer.com.xml]: Please rename to somewhat more generic

Comment (injust) [reply src/chrome/content/rules/Shouldianswer.com.xml]: Fixed, thanks!

File: src/chrome/content/rules/Shouldianswer.com.xml (hunk 1 above)
Comment (injust) [reply src/chrome/content/rules/Shouldianswer.com.xml]: Added, thanks!

**Pull request merged**